src/sysoptions.h

```diff
@@ -208,7 +208,7 @@
 #define DROPBEAR_SK_ED25519 ((DROPBEAR_SK_KEYS) && (DROPBEAR_ED25519))
 #endif
 
-#define DROPBEAR_PQHYBRID DROPBEAR_SNTRUP761
+#define DROPBEAR_PQHYBRID DROPBEAR_SNTRUP761 || DROPBEAR_MLKEM768
```

Comment:
This needs brackets `(DROPBEAR_SNTRUP761 || DROPBEAR_MLKEM768)`

Reply:
Fixed.